app/controllers/comments_controller.rb

```diff
@@ -0,0 +1,54 @@
+class CommentsController < ApplicationController
+  before_action :set_task
+  before_action :set_comment, only: %i[edit update destroy]
+  before_action :authorize_comment, only: %i[destroy]
```

Comment:
Before action uses for don't repeat yourself, but here you use it just for 1 action, this is unnecessary